janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -230,12 +238,17 @@ TableMetadata getTableMetadata(final String name) throws BackendException {
     @Override
     public void close() {
         try {
-            clearJmxMetrics();
-            this.session.close();
-        } finally {
-            if(this.executorService != null && !executorService.isShutdown()){
-                this.executorService.shutdownNow();
+            ExceptionWrapper exceptionWrapper = new ExceptionWrapper();
+            executeWithCatching(this::clearJmxMetrics, exceptionWrapper);
+            executeWithCatching(session::close, exceptionWrapper);
+            if(exceptionWrapper.getThrowable() != null){
+                if(exceptionWrapper.getThrowable() instanceof RuntimeException){
+                    throw (RuntimeException) exceptionWrapper.getThrowable();
```

Comment:
Why not using `throwIfException`?

Reply:
Changed it

Reply:
Diff: https://github.com/JanusGraph/janusgraph/compare/fa1ec0d02376b353e2755e51894adff269299540..c659d549f83389162ed7b9cfb82e845a085513ea